src/Cartography/Cartography.DynamicMap/Core/MapViewPortCoordinates.cs

```diff
@@ -1,70 +1,67 @@
 ﻿using System;
-using GeolocatorService;
 using Uno.Extensions;
-using Windows.Devices.Geolocation;
 
-namespace Cartography.DynamicMap
+namespace Cartography.DynamicMap;
+
+public class MapViewPortCoordinates : IEquatable<MapViewPortCoordinates>
 {
-	public class MapViewPortCoordinates : IEquatable<MapViewPortCoordinates>
+	public MapViewPortCoordinates(Windows.Devices.Geolocation.BasicGeoposition northWest, Windows.Devices.Geolocation.BasicGeoposition northEast, Windows.Devices.Geolocation.BasicGeoposition southWest, Windows.Devices.Geolocation.BasicGeoposition southEast)
```

Comment:
I don't understand those changes. Why remove the using and repeat the namespace everywhere?

Reply:
Existe in Geolocator.

Reply:
What does that mean? How is this related to having the namespace in a using or not?

Reply:
It means there is a conflict otherwise and won't compile, at some place, it also means it's only used for a specific platform, so I'll use the full namespace.